### PR TITLE
tikv: recorrect the descriptions on the feature `region-split-size` part.

### PR DESCRIPTION
**Pull request by LykxSassinator**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Recorrect the previous release notes on the configuration `region-split-size` in tikv.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tune-region-performance.md [hunk 1]
```diff
@@ -17,11 +17,11 @@ TiKV 自动将底层数据进行[分片](/best-practices/tidb-best-practices.md#
 
 ## 使用 `region-split-size` 调整 Region 大小
 
-> **警告：**
+> **注意：**
 >
-> 自定义 Region 大小是在 TiDB v6.1.0 引入的实验特性，不建议在生产环境中配置。使用此特性的风险包括：
+> 使用此特性的风险包括：
```
[the remark this answers is not in the document]
Comment (LykxSassinator) [reply tune-region-performance.md:22]: How about “使用此特性的风险” =》“使用此特性的潜在影响包括”

File: tune-region-performance.md [hunk 2]
```diff
@@ -17,11 +17,11 @@ TiKV 自动将底层数据进行[分片](/best-practices/tidb-best-practices.md#
...
 >
-> + 更容易发生性能抖动。
+> + 可能发生性能抖动。
```
Comment (bufferflies) [inline tune-region-performance.md:24]: 1. 调度更容易触发 flow control， 建议同时调整这个阈值

File: tune-region-performance.md (hunk 1 above)
Comment (tonyxuqqi) [inline tune-region-performance.md:22]: 这里的风险都是调大region-split-size造成的，建议可以显式的说明调大size、调小size的风险和收益。
然后给一个范围，我建议就48MB ~ 256MB。

File: tune-region-performance.md [hunk 3]
```diff
@@ -11,26 +11,26 @@ summary: 了解如何通过调整 Region 大小等方法对 Region 进行性能
 
 TiKV 自动将底层数据进行[分片](/best-practices/tidb-best-practices.md#数据分片)，所有数据按照 key 的范围划分为若干个 Region。当某个 Region 的大小超过一定限制后，TiKV 会将它分裂为多个 Region。
 
-在大量数据的场景下，可能会出现 Region 数量过多，从而带来更多的资源开销和导致[性能回退](/best-practices/massive-regions-best-practices.md#性能问题)的问题。在固定数据量下，Region 越大，则 Region 个数越少。从 v6.1.0 开始，TiDB 支持设置自定义的 Region 大小。Region 默认的大小约为 96 MiB，将其调大可以减少 Region 个数。
+在大量数据的场景下，Region 较小，可能会出现 Region 数量过多，从而带来更多的资源开销和导致[性能回退](/best-practices/massive-regions-best-practices.md#性能问题)的问题。从 v6.1.0 开始，TiDB 支持设置自定义的 Region 大小。Region 默认的大小约为 96 MiB，将其调大可以减少 Region 个数。
```
Comment (qiancai) [inline tune-region-performance.md:14]:
```suggestion
在大量数据的场景下，如果 Region 较小，可能会出现 Region 数量过多的情况，从而带来更多的资源开销和导致[性能回退](/best-practices/massive-regions-best-practices.md#性能问题)的问题。从 v6.1.0 开始，TiDB 支持设置自定义的 Region 大小。Region 默认的大小约为 96 MiB，将其调大可以减少 Region 个数。
```

File: tune-region-performance.md [hunk 4]
```diff
@@ -11,26 +11,26 @@ summary: 了解如何通过调整 Region 大小等方法对 Region 进行性能
...
 
 开启 [Hibernate Region](/best-practices/massive-regions-best-practices.md#方法四开启-hibernate-region-功能) 或 [`Region Merge`](/best-practices/massive-regions-best-practices.md#方法五开启-region-merge) 也可以减少过多 Region 带来的性能开销。
 
 ## 使用 `region-split-size` 调整 Region 大小
 
-> **警告：**
+> **注意：**
 >
-> 自定义 Region 大小是在 TiDB v6.1.0 引入的实验特性，不建议在生产环境中配置。使用此特性的风险包括：
+> Region 大小的推荐值的范围为 [48MiB, 258MiB]，常用大小为 96 MiB、128 MiB、256 MiB。不推荐将 Region 大小设置超过 1 GiB，强烈建议不超过 10 GiB。过大的 Region 可能带来以下影响：
```
Comment (qiancai) [inline tune-region-performance.md:22]:
```suggestion
> Region 大小的推荐范围为 [48MiB, 258MiB]，常用的大小包括 96 MiB、128 MiB、256 MiB。不推荐将 Region 大小设置超过 1 GiB，强烈建议不超过 10 GiB。设置过大的 Region 可能带来以下影响：
```

File: tune-region-performance.md [hunk 5]
```diff
@@ -11,26 +11,26 @@ summary: 了解如何通过调整 Region 大小等方法对 Region 进行性能
...
 >
-> + 更容易发生性能抖动。
+> + 可能发生性能抖动。
```
Comment (qiancai) [inline tune-region-performance.md:24]:
```suggestion
> + 性能抖动。
```

Comment (qiancai) [reply tune-region-performance.md:24]: L22 已经写了“可能“

File: tune-region-performance.md [hunk 6]
```diff
@@ -11,26 +11,26 @@ summary: 了解如何通过调整 Region 大小等方法对 Region 进行性能
...
 
 ## 使用 bucket 增加并发
 
 > **警告：**
 >
 > 当前该功能为实验特性，不建议在生产环境中使用。
 
-Region 调大以后，为了增加查询并发，应当设置 [`coprocessor.enable-region-bucket`](/tikv-configuration-file.md#enable-region-bucket-从-v610-版本开始引入) 为 `true`。这个配置会将每个 Region 划分为更小的区间 bucket，并且以这个更小的区间作为并发查询单位，以提高扫描数据的并发度。bucket 的大小通过 [`coprocessor.region-bucket-size`](/tikv-configuration-file.md#region-bucket-size-从-v610-版本开始引入) 来控制。
\ No newline at end of file
+Region 调大以后，若为了进一步提高查询的并发度，则应当同步设置 [`coprocessor.enable-region-bucket`](/tikv-configuration-file.md#enable-region-bucket-从-v610-版本开始引入) 为 `true`。这个配置会将每个 Region 划分为更小的区间 bucket，并且以这个更小的区间作为并发查询单位，以提高扫描数据的并发度。bucket 的大小通过 [`coprocessor.region-bucket-size`](/tikv-configuration-file.md#region-bucket-size-从-v610-版本开始引入) 来控制。
```
Comment (qiancai) [inline tune-region-performance.md:36]:
```suggestion
Region 调大以后，如需进一步提高查询的并发度，可以设置 [`coprocessor.enable-region-bucket`](/tikv-configuration-file.md#enable-region-bucket-从-v610-版本开始引入) 为 `true`。这个配置会将每个 Region 划分为更小的区间 bucket，并且以这个更小的区间作为并发查询单位，以提高扫描数据的并发度。bucket 的大小通过 [`coprocessor.region-bucket-size`](/tikv-configuration-file.md#region-bucket-size-从-v610-版本开始引入) 来控制。
```

File: tune-region-performance.md [hunk 7]
```diff
@@ -11,26 +11,26 @@ summary: 了解如何通过调整 Region 大小等方法对 Region 进行性能
...
 
 开启 [Hibernate Region](/best-practices/massive-regions-best-practices.md#方法四开启-hibernate-region-功能) 或 [`Region Merge`](/best-practices/massive-regions-best-practices.md#方法五开启-region-merge) 也可以减少过多 Region 带来的性能开销。
 
 ## 使用 `region-split-size` 调整 Region 大小
 
-> **警告：**
+> **注意：**
 >
-> 自定义 Region 大小是在 TiDB v6.1.0 引入的实验特性，不建议在生产环境中配置。使用此特性的风险包括：
+> Region 大小的推荐范围为 [48MiB, 258MiB]，常用的大小包括 96 MiB、128 MiB、256 MiB。不推荐将 Region 大小设置超过 1 GiB，强烈建议不超过 10 GiB。设置过大的 Region 可能带来以下影响：
```
Comment (qiancai) [inline tune-region-performance.md:22]:
```suggestion
> Region 大小的推荐范围为 [48MiB, 258MiB]，常用的大小包括 96 MiB、128 MiB、256 MiB。不推荐将 Region 大小设置超过 1 GiB，强烈建议不超过 10 GiB。过大的 Region 可能带来以下影响：
```

File: releases/release-6.1.0.md [hunk 8]
```diff
@@ -59,9 +59,15 @@ TiDB 版本：6.1.0
 
 ### 性能
 
-* 支持自定义 Region 大小（实验特性）
+* 支持自定义 Region 大小
 
-    设置更大的 Region 可以有效减少 Region 数量，降低 Region 管理成本，提升集群性能和稳定性。该特性引入 bucket 概念，即将每个 Region 划分为更小的区间 bucket。使用 bucket 作为并发查询单位能够优化 Region 调大时的查询性能，动态调整热点 Region 的大小来保证热点调度效率和负载均衡。该特性目前属于实验特性，不建议在生产环境使用。
+    设置更大的 Region 可以有效减少 Region 数量，降低 Region 管理成本，提升集群性能和稳定性。
+
+    [用户文档](/tune-region-performance.md)，[#11515](https://github.com/tikv/tikv/issues/11515)
```
Comment (qiancai) [inline releases/release-6.1.0.md:66]:
```suggestion
    [用户文档](/tune-region-performance.md#使用-region-split-size-调整-region-大小)，[#11515](https://github.com/tikv/tikv/issues/11515)
```

File: releases/release-6.1.0.md [hunk 9]
```diff
@@ -59,9 +59,15 @@ TiDB 版本：6.1.0
...
+
+* 支持 Bucket 功能（实验特性）
+
+    该特性引入 bucket 概念，即将每个 Region 划分为更小的区间 bucket。使用 bucket 作为并发查询单位能够优化 Region 调大时的查询性能，动态调整热点 Region 的大小来保证热点调度效率和负载均衡。该特性目前属于实验特性，不建议在生产环境使用。
```
Comment (qiancai) [inline releases/release-6.1.0.md:70]:
```suggestion
    当 Region 调大以后，为了进一步提高查询的并发度，TiDB 引入 bucket 概念，即将每个 Region 划分为更小的区间 bucket。使用 bucket 作为并发查询单位能够优化 Region 调大时的查询性能，动态调整热点 Region 的大小来保证热点调度效率和负载均衡。该特性目前属于实验特性，不建议在生产环境使用。
```

File: releases/release-6.1.0.md [hunk 10]
```diff
@@ -59,9 +59,15 @@ TiDB 版本：6.1.0
...
 
     [用户文档](/tune-region-performance.md)，[#11515](https://github.com/tikv/tikv/issues/11515)
```
Comment (qiancai) [inline releases/release-6.1.0.md:72]:
```suggestion
    [用户文档](/tune-region-performance.md#使用-region-split-size-调整-region-大小)，[#11515](https://github.com/tikv/tikv/issues/11515)
```

File: releases/release-6.1.0.md [hunk 11]
```diff
@@ -59,9 +59,15 @@ TiDB 版本：6.1.0
 
 ### 性能
 
-* 支持自定义 Region 大小（实验特性）
+* 支持自定义 Region 大小
 
-    设置更大的 Region 可以有效减少 Region 数量，降低 Region 管理成本，提升集群性能和稳定性。该特性引入 bucket 概念，即将每个 Region 划分为更小的区间 bucket。使用 bucket 作为并发查询单位能够优化 Region 调大时的查询性能，动态调整热点 Region 的大小来保证热点调度效率和负载均衡。该特性目前属于实验特性，不建议在生产环境使用。
+    设置更大的 Region 可以有效减少 Region 数量，降低 Region 管理成本，提升集群性能和稳定性。
```
Comment (qiancai) [inline releases/release-6.1.0.md:64]:
```suggestion
    从 v6.1.0 起，你可以通过 [`coprocessor.region-split-size`](/tikv-configuration-file.md#region-split-size) 设置更大的 Region 可以有效减少 Region 数量，降低 Region 管理成本，提升集群性能和稳定性。
```

File: releases/release-6.1.0.md [hunk 12]
```diff
@@ -59,11 +59,17 @@ TiDB 版本：6.1.0
 
 ### 性能
 
-* 支持自定义 Region 大小（实验特性）
+* 支持自定义 Region 大小
 
-    设置更大的 Region 可以有效减少 Region 数量，降低 Region 管理成本，提升集群性能和稳定性。该特性引入 bucket 概念，即将每个 Region 划分为更小的区间 bucket。使用 bucket 作为并发查询单位能够优化 Region 调大时的查询性能，动态调整热点 Region 的大小来保证热点调度效率和负载均衡。该特性目前属于实验特性，不建议在生产环境使用。
+    从 v6.1.0 起，你可以通过 [`coprocessor.region-split-size`](/tikv-configuration-file.md#region-split-size) 设置更大的 Region 可以有效减少 Region 数量，降低 Region 管理成本，提升集群性能和稳定性。
```
Comment (qiancai) [inline releases/release-6.1.0.md:64]:
```suggestion
    从 v6.1.0 起，你可以通过 [`coprocessor.region-split-size`](/tikv-configuration-file.md#region-split-size) 设置更大的 Region 从而有效减少 Region 数量，降低 Region 管理成本，提升集群性能和稳定性。
```

**Review by qiancai — APPROVED** [review]

File: releases/release-6.1.0.md [hunk 13]
```diff
@@ -59,11 +59,17 @@ TiDB 版本：6.1.0
...
-* 支持自定义 Region 大小（实验特性）
+* 支持自定义 Region 大小
 
-    设置更大的 Region 可以有效减少 Region 数量，降低 Region 管理成本，提升集群性能和稳定性。该特性引入 bucket 概念，即将每个 Region 划分为更小的区间 bucket。使用 bucket 作为并发查询单位能够优化 Region 调大时的查询性能，动态调整热点 Region 的大小来保证热点调度效率和负载均衡。该特性目前属于实验特性，不建议在生产环境使用。
+    从 v6.1.0 起，你可以通过 [`coprocessor.region-split-size`](/tikv-configuration-file.md#region-split-size) 设置更大的 Region 从而有效减少 Region 数量，降低 Region 管理成本，提升集群性能和稳定性。
 
-    [用户文档](/tune-region-performance.md)，[#11515](https://github.com/tikv/tikv/issues/11515)
+    [用户文档](/tune-region-performance.md#使用-region-split-size-调整-region-大小)，[#11515](https://github.com/tikv/tikv/issues/11515)
+
+* 支持 Bucket 功能（实验特性）
```
Comment (qiancai) [inline releases/release-6.1.0.md:68]:
```suggestion
* 支持使用 bucket 增加并发（实验特性）
```

**Pull request merged**